OSDOCS-20131: adds QE bug fixes RHCL deployment (#113373)
openshift/openshift-docs: modules/proc-use-coredns-single-cluster.adoc

$ oc apply -f ./coredns-manifests.yaml
----

> . Create a `ConfigMap` to define the authoritative zone for CoreDNS by using the following example. This minimal configuration enables the `kuadrant` plugin and GeoIP features.

Reviewer: ```suggestion
. Create a `ConfigMap` object to define the authoritative zone for CoreDNS by using the following example. This minimal configuration enables the `kuadrant` plugin and GeoIP features.
```
If you're referring to the actual API object.

For production or exact GeoIP routing, mount your licensed MaxMind GeoIP database into the CoreDNS pod and update the filename in the `data.Corefile.geoip` parameter.
====

. Apply the ConfigMap by running the following command:
+
[source,terminal]
----

> $ oc apply -f --context $CTX_PRIMARY

Reviewer 2: Same issue again, missing filename in this command

For production or exact GeoIP routing, mount your licensed MaxMind GeoIP database into the CoreDNS pod and update the filename in the `data.Corefile.geoip` parameter.
====

> . Apply the ConfigMap by running the following command:

Reviewer: ```suggestion
. Apply the `ConfigMap` object by running the following command:
```

Similar to the other comments about ConfigMap.

kuadrant-coredns successfully rolled out
----

> . Create the Kubernetes `Secret` that {prodname} uses to interact with CoreDNS. This secret specifies the zones this provider instance is authoritative for by running the following command:

Reviewer: This wording makes it sound like the secret runs the command. Maybe this would work instead?

```suggestion
. Create the Kubernetes `Secret` that {prodname} uses to interact with CoreDNS. This secret specifies the zones this provider instance is authoritative for. To create the secret, run the following command:
```

+
[source,terminal,subs="+quotes"]
----
$ oc create secret generic coredns-credentials \
  --namespace=_<kuadrant_system>_ \
  --type=kuadrant.io/coredns \
  --from-literal=ZONES="${KUADRANT_SUBDOMAIN}.${ONPREM_DOMAIN}" \
  --context ${CTX_PRIMARY}
----
+
Replace `_<kuadrant_system>_` with the namespace of your {prodname} deployment.

.Verification

* Check the status of the `DNSRecord` CR by running the following commands:
+

> [source,terminal]

Reviewer: ```suggestion
[source,terminal,subs="+quotes"]
```

----
$ oc get dnsrecord _<name>_ -n _<namespace>_ -o jsonpath='{.status.conditions[?(@.type=="Ready")]}'
----
+

> [source,terminal]

Reviewer: ```suggestion
[source,terminal,subs="+quotes"]
```
